From 494e0aa627cfee1ad9e7351a1b8376ad26549d1a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Oliver=20St=C3=A4bler?= <oliver.staebler@bytesatwork.ch>
Date: Mon, 8 Mar 2021 15:29:52 +0100
Subject: [PATCH] board: bytedevkit: Introduce choice for RAM size

A new configuration is introduced to select the different RAM configurations on
the byteENGINE STM32MP1x.
---
 board/bytesatwork/bytedevkit/Kconfig | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/board/bytesatwork/bytedevkit/Kconfig b/board/bytesatwork/bytedevkit/Kconfig
index b3c53d6ba0..b8fc5dc1ef 100644
--- a/board/bytesatwork/bytedevkit/Kconfig
+++ b/board/bytesatwork/bytedevkit/Kconfig
@@ -9,6 +9,18 @@ config SYS_VENDOR
 config SYS_CONFIG_NAME
 	default "bytedevkit"
 
+choice
+	prompt "byteENGINE STM32MP1 DDR size"
+	default BYTEENGINE_STM32MP1_RAM_512
+
+config BYTEENGINE_STM32MP1_RAM_512
+	bool "512MB"
+
+config BYTEENGINE_STM32MP1_RAM_1024
+	bool "1024MB"
+
+endchoice
+
 source "board/st/common/Kconfig"
 
 endif
-- 
2.20.1

